schema.sql

```diff
@@ -3,3 +3,12 @@
 CREATE TABLE animals (
     name varchar(100)
 );
```

Comment:
- [ ] There should be only one `animals` table for the schema, your code right now gives an impression of having two animals tables, so consider removing lines 3 to 5.